src/config/dynamic-theme-fixes.config

```diff
@@ -2425,7 +2426,16 @@ CSS
 .justify-content-center.align-items-center.flex-lg-column.flex-md-row.card-2.card > .card-body > p,
 .justify-content-center.align-items-center.flex-lg-column.flex-md-row.card-3.card > .card-body > p,
 .js-hiring-open-pos.join__btn.mt-5.btn.col-sm-auto.col{
-  color: var(--darkreader-neutral-background) !important;
+    color: var(--darkreader-neutral-background) !important;
+}
+
+.logo_homepage {
+    background-image: url("https://duckduckgo.com/assets/logo_homepage.alt.v108.svg") !important;
+}
+
+.header--html .header__logo, .header__logo {
+    background: no-repeat center url("https://duckduckgo.com/assets/logo_header.alt.v108.svg") !important;
+    background: no-repeat center/auto 36px url("https://duckduckgo.com/assets/logo_header.alt.v108.svg") !important;
```

Comment:
Once again, why double property here?

Reply:
Don't mark it as resolved when I still don't understand why a double property is here.

Reply:
There was a double property there because I "yoinked" the css from ddg 
> and yes I did yoink the CSS

Reply:
You can remove the top property it doesn't add anything to the fix.